build/dockerfiles/rhel.Dockerfile

```diff
@@ -20,15 +21,21 @@ RUN chmod +x /usr/share/container-scripts/httpd/pre-init/40-ssl-certs.sh && \
 RUN yum update -y systemd && yum clean all && rm -rf /var/cache/yum && \
     echo "Installed Packages" && rpm -qa | sort -V && echo "End Of Installed Packages"
 
-# BEGIN these steps might not be required
+RUN echo "<FilesMatch "\""^\\.ht"\"">" >> /etc/httpd/conf/httpd.conf && \
+    echo "Require all denied" >> /etc/httpd/conf/httpd.conf && \
+    echo "</FilesMatch>" >> /etc/httpd/conf/httpd.conf
+
+RUN sed -i /etc/httpd/conf.d/ssl.conf \
```

Comment:
hello, could you describe what it does fix ?
AFAIK ssl engine is not enabled (it is listening on http port)

Reply:
Downstream SSL is used and IBM security flagged that the encryption strength could be better. Thus the entries for the Cipher Suite and Protocol.

Reply:
@ArvinB from what I see there, ssl is started, but the 8080 port is used and not the 8043
so either the issue is to not start ssl port or then it needs to use 8043 port in deployments (which is not the case in operator or in helm charts)

Reply:
@benoitf Yes, I see your point and also see port 8080 is used. Even if the ssl port (8043) is not started. We have guidance to still setup various security provisions within the config for any container image that IBM publishes. That way if the service is turned on and used, then it is already secured.

So I am requesting please if we can keep this config upstream here, because it will make it's way downstream into an IBM published container image.

Reference from OpenShift:
![image](https://user-images.githubusercontent.com/1202744/127680489-f99cd9fd-4f23-4eb8-8884-3a522e688e0a.png)

Reply:
I would say it's misleading to say it's solving security issues that we don't have as we never use that part.
Why are we even copy ssl config file if it's never used.

In case you really want to keep such useless configuration, I would recommend to avoid any sed expression that can break at any time silently. For example SSLProtocol property entry can be updated and you think sed is replacing but it may not.

So probably sed expression need to be more robust if you want to keep it and check that it's correctly applied.

Reply:
Hi @benoitf 

I can agree with that, it is misleading to say that we are solving a security issue. As the PR title says, it's more accurate to simply say that security updates are being contributed into the httpd configs.

We do want to keep the config. So we can put something else in place instead of using a `sed` expression. I am open to an alternative recommendation? However, using `sed` is pretty common and well used all the time. Is the concern that there isn't a check in place to ensure the settings have been applied properly?

Reply:
@ArvinB the sed expression you used assume to replace a property + a default value so a safer sed expression would be to replace the value whatever is the default value is

And yes it would be great to ensure the property has been applied

Reply:
I will do this at the end of next week. Thank you for the quick response.

Reply:
@benoitf 
Sorry for the delay, work and life has been nutz!
I just pushed changes based on our convo.